tests/test_route.py

```diff
@@ -0,0 +1,47 @@
+from urllib.request import urlopen
+
+import pytest
+from flask import url_for
+
+
+@pytest.mark.usefixtures("live_server")
+class TestNotificationEndpoint:
+    def test_notification_route_response(self):
+        """
+        GIVEN: function runs to connect with the given route/endpoint.
```

Comment:
The GIVEN part should describe the state before this test runs. i.e. what's presumed to be existing. Like, "given the flask application is up and running"

Reply:
UPDATED